pdfplumber/pdf.py

```diff
@@ -33,10 +33,10 @@ def __init__(self, stream, pages=None, laparams=None, precision=0.001, password=
                 self.metadata[k] = list(map(decode_text, v))
             elif isinstance(v, PSLiteral):
                 self.metadata[k] = decode_text(v.name)
-            elif isinstance(v, bool):
-                self.metadata[k] = v
-            else:
+            elif isinstance(v, str) or isinstance(v, bytes):
```

Comment:
I think this can be slightly simpler: `elif isinstance(v, (str, bytes)):`

Reply:
Thanks for sharing this @jsvine I didn't know you could specify a tuple in `isinstance()`. Learnt something new today :)